samples/daily-prebuilt/permissions-can-admin/src/daily.js

```diff
@@ -31,7 +31,9 @@ const createToken = async ({ isOwner, roomName }) => {
   return tokenData.token;
 };
 
-export const api = {
+const api = {
   createToken,
   createRoom,
 };
+
+export default api;
```

Comment:
Related to default export being preferred when it is the only thing being exported